### PR TITLE
New actions added

### PR DESCRIPTION
**Pull request by davidpa86**

### CODE REVIEW

**Review by Souler — CHANGES_REQUESTED** [review]
the .DS_Store shouldn't be here

File: sourcemaps-to-datadog/action.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  using: composite
+
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Upload sourcemaps
+      env:
+        DATADOG_SITE: "datadoghq.eu"
+        DATADOG_API_HOST: "api.datadoghq.eu"
```
[the remark this answers is not in the document]
Comment (davidpa86) [reply sourcemaps-to-datadog/action.yml]: Done

File: sourcemaps-to-datadog/action.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    type: string
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
```
[the remark this answers is not in the document]
Comment (davidpa86) [reply sourcemaps-to-datadog/action.yml]: Done

File: sourcemaps-to-datadog/action.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+name: Upload source maps to Datadog
+description: Upload your source maps to Datadog to de-obfuscate your different stack traces. For any given error, you can access the file path, line number, and code snippet for each frame of the related stack trace.
```
[the remark this answers is not in the document]
Comment (davidpa86) [reply sourcemaps-to-datadog/action.yml]: Done

**Review by Souler — APPROVED** [review]

**Pull request merged**